content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -109,6 +110,34 @@ Each HTTP rule contains the following information:
 A default backend is often configured in an Ingress controller to service any requests that do not
 match a path in the spec.
 
+### Path Types
+
+Each path in an Ingress has a corresponding path type. There are three supported
+path types:
+
+* **ImplementationSpecific:** With this default path type, matching is up to the
+  IngressClass. Implementations can treat this as a separate PathType or treat
+  it identically to Prefix or Exact path types
+
+* **Exact:** Matches the URL path exactly and with case sensitivity.
+
+* **Prefix:** PathTypePrefix matches based on a URL path prefix split by `/`.
```

Comment:
```suggestion
* **Prefix:** Prefix matches based on a URL path prefix split by `/`.
```

Reply:
PathTypePrefix is the internal Go type, but Prefix should be the string specified for the pathType value

Reply:
Good catch, thanks! Should be fixed now. Also I don't think I can mark things as resolved on this PR, so feel free to mark this or anything else as resolved if it seems like it's been covered.